### DISCUSSION TITLE
How Do I Fix Update of uYou?

### DISCUSSION THREAD

**Discussion by Metercedes** [qnblackcat/uYouPlus#1056] [Q&A] [answered]
I don't know the reason for this problem, but the more I think about it, the more it gets annoying. I download the latest version of [uYouPlus](https://github.com/qnblackcat/uYouPlus/releases) from GitHub and install it with the AltStore, and no matter what I do, there is never an up-to-date version. When I press the _'Update Now'_ button, it goes to another [project ](https://miro92.com/repo/depictions/?p=com.miro.uyou) and it is not that related to uYouPlus. They are old. How do I update this to **v2.3~1**? I've searched a bit but no one ever mentioned this. Perhaps I am missing something? But what?

**Extra Question:** How do I hide uYou button on the main menu of YouTube? _(In the hotbar of YouTube)_

**Dear developers, please do not remove this post. Instead please show patience and explain the situation in a short way. Thank you! :)**

Attachments (Screenshots):

![IMG_0244](https://user-images.githubusercontent.com/51666146/230185341-d58e1d60-e1a3-40a0-8ac9-e3591deee62d.PNG)

![IMG_0243](https://user-images.githubusercontent.com/51666146/230185345-5ca6436f-95a4-484b-b06e-0e364e59aa07.PNG)

**Comment by qnblackcat** [comment, marked as answer]
https://www.reddit.com/r/sideloaded/comments/11s3vof/uyouplus_installation_question/jcflsn4/
